models/daily_spend.sql

```diff
@@ -81,7 +81,7 @@ storage_terabytes_daily as (
     left join storage_terabytes_daily on dates.date = storage_terabytes_daily.date
     left join {{ ref('daily_rates') }} on
         storage_terabytes_daily.date = daily_rates.date
-        and daily_rates.usage_type = 'storage'
+        and (daily_rates.usage_type = 'storage' or daily_rates.usage_type = 'overage-storage')
```

Comment:
We can either do a group by here (by date, service, storage type, warehouse_name, database_name), or do it in a CTE just on date and join it in.

Reply:
I've added the group by (like the other spend types) and the models have run successfully.  Full disclosure, we have no storage spend of these types and no tests so feels like a bit of a hit and hope commit.  https://github.com/aphethean1/dbt-snowflake-monitoring/commit/17379120f6ecafce9eaf7a686dbe88df1b69c06e